### ISSUE TITLE
Region change problems

### ISSUE THREAD

**Issue by FINDarkside** [FINDarkside/TLD-Save-Editor#35]
Sometimes when you change region you can't enter building anymore. Every time you enter a building you'd just fall trough the map.

Sometimes changing region from game put you to "fresh state" region even if you have been in that region previously.

**Comment by FINDarkside** [comment]
[Example save](https://1drv.ms/u/s!AumyQUD3NBRWyEqDuDzQaXUzoJFC?e=l4i4Ql), entering the house in front does some wonky stuff.

**Comment by Pt-Djefferson** [comment]
`m_ForceNextSceneLoadTriggerScene` must be nulled after changing region.
`Global.GameManagerData.SceneTransition.m_ForceNextSceneLoadTriggerScene = null;`

**Comment by FINDarkside** [comment]
Great this seems to fix the falling trough map at least! Not sure if this fixes the alternative reality bug as well.

**Comment by Pt-Djefferson** [comment]
Dunno how to help with alternative reality. Causes  of "fresh state" can only be in corrupted, deleted or outdated m_Dict maps sections.

**Comment by FINDarkside** [comment]
Well there's a bit more to it. The game actually has multiple scenes of the same region (sandbox, story and so on) and first I suspected this is the problem. At one point I supposedly fixed it with [these 2 lines](https://github.com/FINDarkside/TLD-Save-Editor/blob/master/The%20Long%20Dark%20Save%20Editor%202/GameSave.cs#L58). Without these, the game was actually showing you the items of your old region when you were actually in completely different region. Or something like that, I don't remember anymore. But yeah, now that I checked the last report of this "alternate dimension" it included this falling thing so probably caused by this. I'll create a new issue if someone still has the problem.
